### ISSUE TITLE
How to deal with missing or empty point in a series?

### ISSUE THREAD

**Issue by NintendoPlayer** [Live-Charts/Live-Charts#746]
Instructions: replace text surrounded by '*' with your case, please delete this line.

#### How to reproduce?

I have a datalogging system that saves values to a DB.  I have a calendar control, the user picks a day, and I load the data from a DB between startdate and enddate.  I have a chart with datetime formatting.  Everything is working correctly, livecharts is awesome.

But I have one issue.  I don't have consecutive values, so I could have 20 or 30 points from 17:30 to 17:50, and some other 20 o 30 values from 20:00 to 20:35. I don't have data between 17:50 and 20:00 but, obvioulsy, a line is getting plotted between those two points.  Is there a way to tell the chart that there is no data between those two points so I don't want a line to be drawn between them?  Or some way to tell that between those two points is something like "missing data", so a dashed line can be drawn?

#### I am using

<table>
<tr>
    <td>Live-Charts version</td>
    <td>0.9.7.0</td>
</tr>
<tr>
    <td>.Net Version</td>
    <td>4.7.1</td>
</tr>
<tr>
    <td>Windows</td>
    <td>Win 10 Pro x64</td>
</tr>
</table>

**Comment by ZombiesWithCoffee** [comment]
Use double.NaN when you don't know a value in a line series, that point will be undefined and will be ignored when the series is drawn.

https://lvcharts.net/App/examples/v1/wpf/Missing%20Points

**Comment by NintendoPlayer** [comment]
Yeah, but that way I have to create empty points between the two points.  That's what I did when I was using Microsoft Charting.  I wanted to prevent filling the chart with empty points

**Comment by ZombiesWithCoffee** [comment]
You can fill your chart with the known points, then use a LabelFormatter method to supply your axis text values.

            <wpf:CartesianChart.AxisX>
                <wpf:Axis LabelFormatter="{Binding DateFormatter}" />
            </wpf:CartesianChart.AxisX>

**Comment by NintendoPlayer** [comment]
LabelFormatter is already set as datetime.  This is working well, it's not related to the issue.

I solved it somehow:  The data has a fixed distance between points, 1 second, 1 minute, 30 minutes, etc.  So I check the distance between the first two pints, and the check each point with the previous one.  If the timespan diference of the pair of points is greater thant the distance calculated of the first two points, y add a double.Nan point to the series.

That way I don't get a line between the two separated points.  But it would be useful to have some sort of EmptyPointStyle property that changes the style of the series when there is an empty point between two points with data.

**Comment by JDiLenarda** [comment]
Though OP has certainly moved away since, here’s my take.
I faced the same problem and considered the same solution, but finally I went for splitting my series into several others and binding a `CollectionSeries` rather a single `Series`. I think it’s cleaner and easier than adding virtual values, and values without neighbours still appears without any trick.
